packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -320,11 +422,27 @@ function getFaucetHost(client: Client): FaucetNetwork | undefined {
   throw new XRPLFaucetError('Faucet URL is not defined or inferrable.')
 }
 
+/**
+ * Get the faucet pathname based on the faucet hostname.
+ *
+ * @param hostname - hostname.
+ * @returns A String with the correct path for the input hostname.
+ * If hostname undefined or cannot find (key, value) pair in {@link FaucetNetworkPaths}, defaults to '/accounts'
+ */
+function getFaucetPath(hostname: string | undefined): string {
```

Comment:
```suggestion
function getDefaultFaucetPath(hostname: string | undefined): string {
```
(This only works for known addresses and ones following the default format, so I think it'd be clearer to add that to the function name)